kitsune/products/tests/test_templates.py

```diff
@@ -158,3 +159,52 @@ def test_subtopics(self):
         eq_(200, r.status_code)
         pqdoc = pq(r.content)
         eq_(1, len(pqdoc('li.subtopic')))
+
+    def test_community_support_shown_for_default_language(self):
+        """Verify the get community support is shown in the topics for en-US."""
+        p = ProductFactory()
+        d = DocumentFactory(slug='get-community-support', locale=settings.LANGUAGE_CODE)
```

Comment:
Right, that was testing the previous behavior with checking, if the document even exists.